internal/pkg/types/atto_fil.go

```diff
@@ -130,105 +98,22 @@ func NewAttoFILFromFILString(s string) (AttoFIL, bool) {
 
 // NewAttoFILFromString allocates a new AttoFIL set to the value of s attofilecoin,
 // interpreted in the given base, and returns it and a boolean indicating success.
-func NewAttoFILFromString(s string, base int) (out AttoFIL, err bool) {
-	_, err = out.val.SetString(s, base)
-	return
-}
-
-// Add sets z to the sum x+y and returns z.
-func (z AttoFIL) Add(y AttoFIL) (out AttoFIL) {
-	out.val.Add(&z.val, &y.val)
-	return
-}
-
-// Sub sets z to the difference x-y and returns z.
-func (z AttoFIL) Sub(y AttoFIL) (out AttoFIL) {
-	out.val.Sub(&z.val, &y.val)
-	return
-}
-
-// MulBigInt multiplies attoFIL by a given big int
-func (z AttoFIL) MulBigInt(x *big.Int) (out AttoFIL) {
-	out.val.Mul(&z.val, x)
-	return
+func NewAttoFILFromString(s string, base int) (AttoFIL, bool) {
+	out := specsbig.NewInt(0)
+	_, isErr := out.Int.SetString(s, base)
+	return out, isErr
 }
 
 // DivCeil returns the minimum number of times this value can be divided into smaller amounts
 // such that none of the smaller amounts are greater than the given divisor.
 // Equal to ceil(z/y) if AttoFIL could be fractional.
 // If y is zero a panic will occur.
-func (z AttoFIL) DivCeil(y AttoFIL) AttoFIL {
-	value, remainder := big.NewInt(0).DivMod(&z.val, &y.val, big.NewInt(0))
+func DivCeil(z, y AttoFIL) AttoFIL {
```

Comment:
Assuming we still need this for something, I'd welcome a `DivMod` method in specs-actors. But this looks to be unused.